app/survey_config/business_config.py

```diff
@@ -55,10 +56,21 @@ def __post_init__(self):
         ]
 
     def get_service_links(
```

Comment:
The method is very crowed, new line separators would help a lot with readability.

Reply:
Added bunch of new line separators.